subsys/usb/device_next/class/Kconfig.cdc_ncm

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
+
+config USBD_CDC_NCM_CLASS
+	bool "USB CDC NCM implementation [EXPERIMENTAL]"
+	default y
+	depends on NET_L2_ETHERNET
+	depends on DT_HAS_ZEPHYR_CDC_NCM_ETHERNET_ENABLED
+	select EXPERIMENTAL
+	help
+	  USB CDC Network Control Model (NCM) implementation
+
+if USBD_CDC_NCM_CLASS
+
+config USBD_CDC_NCM_MAX_DGRAM_PER_NTB
+	int "Max number of received datagrams per NTB"
+	range 0 $(UINT16_MAX)
+	default 2
+	help
+	  How many datagrams we are able to receive per NTB.
+
+config USBD_CDC_NCM_SUPPORT_NTB32
+	bool "Support NTB32 format"
+	help
+	  Enable support for NTB32 format which allows larger
+	  packet sizes.
+
+module = USBD_CDC_NCM
+module-str = usbd cdc_ncm
+default-count = 1
+source "subsys/logging/Kconfig.template.log_config"
+rsource "Kconfig.template.instances_count"
```

Comment:
```
module = USBD_CDC_NCM
module-str = usbd cdc_ncm
source "subsys/logging/Kconfig.template.log_config"
```

Only logging template is needed. Other template is not used, it seems to be a copy&paste error in other CDC implementations (my fault).